commonjs/tabs/tabs.js

```diff
@@ -130,6 +151,9 @@ Tabs.prototype = {
         statistics.trackView(document.location.href + '#tab' + (idx + 1));
 
         this._activeTabIdx = idx;
+
+        // if this tab uses a hashPrefix to be recognized from the URL, set the new value.
+        window.location.hash = this.config.hashPrefix + ':' + idx;
```

Comment:
1. hashPrefix must be optional

2. setting the hash creates a history entry, history.back() will have no effect though.

either:

- use location.replace to avoid creating a history entry
- use window.onhashchange to support navigating in tabs